### DISCUSSION TITLE
Repo Size

### DISCUSSION THREAD

**Discussion by Newer1107** [prasanthrangan/hyprdots#237] [Ideas]
you might want to look into this:
![image](https://github.com/prasanthrangan/hyprdots/assets/116626403/4f6d93fa-bf2f-4afd-8922-36000f550848)

the repo size is about 500MB now, I'm not sure what's in there but just cloning and filling up 500MB seems too much, why not make the script fetch stuff instead of me just download 500MB of data?

**Comment by Senshi111** [comment]
Most of those large files are in swww folder in .config because of themes. I think swww folder is about 350 mb or something

**Reply by prasanthrangan** [reply]
Yup, you are right.. it's mainly the wallpapers that takes a lot of space!

**Reply by jpx40** [reply]
maybe we could zip them. and unpack them at installation.
